ansible/roles/ooni-backend/templates/update_bouncer.py.j2

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+import yaml
+import json
+import urllib
+
+MLAB_NS_QUERY_URL = "{{ mlab_ns_query_url_ansible }}"
```

Comment:
I think it's safe to keep this value hardcoded and not require configuring it from ansible as it's probably not going to be changing that often.